app/assets/stylesheets/application.css.scss

```diff
@@ -82,3 +82,28 @@ body {
 .company-id {
   width: 45px;
 }
+
+.panel-primary > .panel-heading {
+  color: #147BD5;
+}
+
+
+.panel-heading h3 {
+  color: white;
+}
+
+.company-data h3 {
+  color: #147BD5;
+}
+
+.companies-by-city h3 {
+  color: #147BD5;
+}
+
+.company-statistics {
+  margin-top: 75px;
+}
+
+.admin-heading h3 {
+  padding-left: 10px;
+}
```

Comment:
I like that the class names are good descriptions of what they cover! :sparkles: